src/contracts/Protocol/BountyQuestion.sol

```diff
@@ -9,63 +9,40 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
 import "./modifiers/OnlyAPI.sol";
 
-// TODO remove ERC721 stuff here
-// TODO introduce a Challenge data model
-// Question -> Challenge is many -> one mapping
-// auto-transition from question -> challenge on:
-//   Voting threshold -> do we want manual approval? -> yes
-//   Admin approval (Messari group wants to write their own challenges)
-// TODO ability for author to unlock and "kill" their question -> reimburse voters
-
-// philosphy -> start out gated and the protocol can evolve in the same way the dao does
-
 /// @custom:security-contact contracts@metricsdao.xyz
-contract BountyQuestion is ERC721, ERC721Enumerable, ERC721URIStorage, ERC721Burnable, Ownable, OnlyApi {
+contract BountyQuestion is Ownable, OnlyApi {
     using Counters for Counters.Counter;
 
     Counters.Counter private _tokenIdCounter;
-    mapping(address => QuestionData[]) public authors; // TODO if we want questions to be transferable, then owner != author
-    mapping(uint256 => uint256) private _createdAt;
 
-    constructor() ERC721("MetricsDAO Question", "MDQ") {
+    // This maps the author to the list of token IDs they have created
+    mapping(address => uint256[]) public authors;
+
+    // This maps the token ID to the question data
+    mapping(uint256 => QuestionData) public questions;
+
+    constructor() {
         _tokenIdCounter.increment();
     }
 
-    // working standard metadata format:  Title, Description, Program
-    function safeMint(address to, string calldata uri) public onlyApi returns (uint256) {
+    function createQuestion(address author, string calldata uri) public onlyApi returns (uint256) {
         uint256 tokenId = _tokenIdCounter.current();
```

Comment:
these are no longer tokens - think about renaming to fakeNFT or just questionCreatedInTime

Reply:
I mean they still all have a unique ID?